drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandler.java

```diff
@@ -141,6 +142,7 @@ public void execute() {
                                      busyIndicatorView.hideBusyIndicator();
                                      presenter.complete();
                                      notifySuccess();
+                                     newResourceSuccessEvent.fire( new NewResourceSuccessEvent( path ) );
```

Comment:
This 'NewResourceSuccessEvent' idea is to only be fired only for the logged user modifications in the workbench, so we can make updates in the UI when the user creates a new asset (e.g. update the breadcrumb when navigating to the recently added asset).